src/Contract/PhpParser/Node/StmtsAwareInterface.php

```diff
@@ -10,6 +10,6 @@
 /**
  * @property Stmt[]|null $stmts
  */
-interface StmtsAwareInterface extends Node
+interface ContainsStmts extends Node
```

Comment:
this seems caused by search/replace, need to be reverted as cause interface conflict.